ClosedXML_Tests/Excel/CalcEngine/MathTrigTests.cs

```diff
@@ -139,5 +140,37 @@ public void SumProduct()
                 Assert.Throws<NoValueAvailableException>(() => ws.Evaluate("SUMPRODUCT(A1:A10, B1:B5)"));
             }
         }
+
+        [TestCase(1, 0.850918128)]
```

Comment:
For the tests, please follow the same pattern as the testing of other functions, where possible, i.e. try to use ` XLWorkbook.EvaluateExpr`

Reply:
yes. will do. Took the wrongest possible example for reference I guess, currently implementing DECIMAL with the other pattern already ;)